components/brave_ads/browser/ads_service_factory.cc

```diff
@@ -103,6 +104,9 @@ void AdsServiceFactory::RegisterProfilePrefs(
   #endif
 
   registry->RegisterIntegerPref(prefs::kBraveAdsIdleThreshold, 15);
+  registry->RegisterBooleanPref(prefs::kBraveAdsShowAdsNotification, true);
+  registry->RegisterTimePref(
+      prefs::kBraveAdsLaunchNotificationTimestamp, base::Time());
```

Comment:
should set to base::Time::Now() instead of base::Time() as this is a timestamp for when a user upgraded

Reply:
done